src/EnergyPlus/EconomicTariff.cc

```diff
@@ -482,11 +485,67 @@ namespace EconomicTariff {
 				tariff( iInObj ).energyConv = 9.4781712e-9;
 				tariff( iInObj ).demandConv = 0.00003412;
 			} else {
-				tariff( iInObj ).convChoice = conversionKWH;
-				tariff( iInObj ).energyConv = 0.0000002778;
-				tariff( iInObj ).demandConv = 0.001;
-				ShowWarningError( RoutineName + CurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\" invalid data" );
-				ShowContinueError( cAlphaFieldNames( 3 ) + "=\"" + cAlphaArgs( 3 ) + "\", Defaulting to KWH." );
+
+
+				// TODO: is this needed?
```

Comment:
Since the default varies depending on the resource type, then I'm ok with a warning that says what conversion it defaulted to.  However, we should be adding M3 or some equivalent to the list of predefined choices.  There is also the problem that water could be billed in CCF or MCF, but currently these choices assume natural gas as some (unstated) heating value.  So, perhaps these should be changed to CCFNaturalGas, and MCFNaturalGas, and then add CCFH2O and MCFH20 and M3H20?   Seem messy, but unless we add a new field to declare the resource type, then CCF is too ambiguous.  @JasonGlazer should chime in here.